audiofile/core/io.py

```diff
@@ -144,6 +144,13 @@ def read(
         >>> signal, sampling_rate = read('stereo.wav', duration=0.1)
         >>> signal.shape
         (2, 800)
+        >>> # Use audresample for resampling and remixing
+        >>> import audresample
+        >>> target_rate = 16000
+        >>> signal = audresample.resample(signal, sampling_rate, target_rate)
```

Comment:
Maybe we should be a little bit more verbose here:

```suggestion
        >>> signal = audresample.resample(signal, sampling_rate, target_rate)
        >>> signal.shape
        (2, 1600)
```